connectors/src/connectors/github/temporal/cast_known_errors.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  ActivityExecuteInput,
+  ActivityInboundCallsInterceptor,
+  Next,
+} from "@temporalio/worker";
+
+export class GithubCastKnownErrorsInterceptor
```

Comment:
What's the purpose?

Reply:
For now, nothing because all errors go into unhandled error, but that's where I'll catch & cast errors

Reply:
roger